apps/backend/src/auto-migrations/index.tsx

```diff
@@ -1,8 +1,17 @@
+import { sqlQuoteIdent } from '@/prisma-client';
 import { Prisma, PrismaClient } from '@prisma/client';
-import { MIGRATION_FILES } from './migration-files';
+import { MIGRATION_FILES } from './../generated/migration-files';
 
+const MIGRATION_LOCK_ID = 59129034;
```

Comment:
can we give this a more readable name?
```suggestion
const MIGRATION_LOCK_ID = "stack_migration_lock";
```